rest-api-spec/src/main/resources/rest-api-spec/api/create.json

```diff
@@ -27,6 +27,12 @@
           "type" : "string",
           "description" : "Sets the number of shard copies that must be active before proceeding with the index operation. Defaults to 1, meaning the primary shard only. Set to `all` for all shard copies, otherwise set to any non-negative value less than or equal to the total number of copies for the shard (number of replicas + 1)"
         },
+        "op_type": {
```

Comment:
I'm in two minds documenting this parameter on `_create`, while perhaps technically correct it makes little sense to ever send `_create?optype=index`.

Reply:
yea, let's not do this. `create` is a different api in the spec that implicitly has `optype` set to `create`. `optype` should not be exposed then.

Reply:
I agree that `_create?op_type=index` does not make any sense, but the server will accept it.
@javanna do you mean that `rest-api-spec` for `_create` should not expose the `op_type` or that the rest layer should validate that if an ` op_type` is sent, then it must be `create`?

Reply:
I mean that the spec for create should not expose op_type. The REST layer could validate the op_type as well, that would be a must. I think we currently replace any existing op_type though.

Reply:
@Mpdreamz can you have another look?